Monofoxe/Monofoxe.Pipeline/Tiled/MapParser.cs

```diff
@@ -48,5 +56,21 @@ public static TiledMap Parse(XmlDocument xml)
 
 			return map;
 		}
+
+		private static Color HtmlHexToColor(string hex)
+		{
+			bool withAlpha = hex.Length > 7;
+			int length = withAlpha ? 4 : 3;
```

Comment:
You don't take into account that color can also be represented as #RGB/#RGBA, not just #RRGGBB/#RRGGBBAA.

Reply:
Actually, investigate if Tiled even uses #RGB/#RGBA format. Is it doesn't then you don't need to implement it.

Reply:
I checked, the format is always `#AARRGGBB` or `#RRGGBB`.

Reply:
And it does!
![image](https://user-images.githubusercontent.com/9446588/95493313-4db7b300-096a-11eb-992f-265be3c367e1.png)